### ISSUE TITLE
UI dark mode: Low contrast for the color red in rustc error messages

### ISSUE THREAD

**Issue by fmease** [rust-lang/rust-playground#1088]
This concerns elements with CSS classes `token error` in the dark UI mode.

![Image](https://github.com/user-attachments/assets/c892b3f4-1e5b-49bd-aa01-2de585fdeb9e)

Contrast between foreground color `#BF1B1B` and background color `#343434` is 2:1 thus failing WCAG AA (according to [WebAIM](https://webaim.org/resources/contrastchecker/?fcolor=BF1B1B&bcolor=343434)).

I haven't checked the other colors that may appear in rustc's diagnostics.

**Edit**: I actually forgot to say explicitly that I personally find it hard to read. The numbers were just meant to back stuff up ^^

**Comment by fmease** [comment]
Ah, I've just seen https://github.com/rust-lang/rust-playground/pull/1087. However, it doesn't seem to address this specific issue (yet).

**Comment by shepmaster** [comment]
> it doesn't seem to address this specific issue

not directly, but it *does* change the output background to #191b1c

![Image](https://github.com/user-attachments/assets/d5d397f5-aada-469d-a534-e4325c40318d)

Thoughts?

Chrome (with [Advanced Perceptual Contrast Algorithm (APCA)](https://developer.chrome.com/blog/new-in-devtools-89/#apca) enabled) does not complain about the error text now (while it does for many other things, such as the warning text)

**Comment by fmease** [comment]
That's a lot more legible 👍 I guess you can mark your PR as fixing this issue.

Still, as alluded to, colors other than red (like yellow & purple) should pop a little bit more when the UI is set to dark.
